test/unittest/Database/DatabaseTests.cpp

```diff
@@ -2100,6 +2111,143 @@ TEST_F(database_tests, get_entity_no_existing)
     ASSERT_THROW(db.get_entity(EntityId()), BadParameter);
 }
 
+TEST_F(database_tests, get_entities_by_name_host)
+{
+    auto hosts = db.get_entities_by_name(EntityKind::HOST, host_name);
+    EXPECT_EQ(hosts.size(), 1);
+    EXPECT_FALSE(hosts[0].first.is_valid());
+    EXPECT_EQ(hosts[0].second, host_id);
+}
```

Comment:
I'm missing tests where there is more than one entity, checking that we do not return just the first one we found.

Reply:
Done for the ones in which a second entity with the same name can be inserted